v0.5/classification_and_detection/tools/accuracy-coco.py

```diff
@@ -27,6 +27,7 @@ def get_args():
     parser.add_argument("--verbose", action="store_true", help="verbose messages")
     parser.add_argument("--output-file", default="coco-results.json", help="path to output file")
     parser.add_argument("--use-inv-map", action="store_true", help="use inverse label map")
+    parser.add_argument("--remove-48-empty-images", action="store_true", help="use inverse label map")
```

Comment:
Help message seems to be incorrect?

Reply:
Right, sorry. I just updated the massage